technical/indicators/overlap_studies.py

```diff
@@ -83,7 +83,7 @@ def hull_moving_average(dataframe, period, field="close") -> ndarray:
 
 
 def vwma(df, window):
-    return (df["close"] * df["volume"]).rolling(window).sum() / df.volume.rolling(window).sum()
+    return ((df["price"] if "price" in df else df["close"]) * df["volume"]).rolling(window).sum() / df.volume.rolling(window).sum()
```

Comment:
The default should clearly remain "close". 
the existence of "price" doesn't automatically mean a user wants to switch to that series.

To have this somewhat aligned, it'll be best to have a new argument in the function - e.g. `field="close"`  - which is then used for dataframe access.

Reply:
Okay, I'll improve it.